plugins/ipc/views.py

```diff
@@ -0,0 +1,126 @@
+"""
+Views for the IPC module
+"""
+import collections
+import datetime
+
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic import TemplateView
+
+from plugins.admissions.models import Encounter, UpstreamLocation
+
+from plugins.ipc import lab, models, utils
+
+
+class IPCHomeView(LoginRequiredMixin, TemplateView):
+    template_name = 'ipc/home.html'
+
+    def get_context_data(self, *a, **k):
+        context = super().get_context_data(*a, **k)
+
+        alerts  = models.InfectionAlert.objects.filter(seen=False).order_by('-trigger_datetime')
+        context['alerts'] = alerts
+
+        monthly = collections.defaultdict(lambda: collections.defaultdict(int))
+
+        today = datetime.date.today()
+        start = datetime.date(today.year-2, today.month, 1)
+
+        for alert in models.InfectionAlert.objects.filter(trigger_datetime__gte=start):
+            key = datetime.date(alert.trigger_datetime.year, alert.trigger_datetime.month, 1)
+            monthly[key][alert.category] += 1
+
+        ticks  = ['x'] + ['{0}-{1:02d}-01'.format(d.year, d.month) for d in sorted(monthly.keys())[-12:]]
+        cdiffs = ['C. Diff'] + [monthly[k][models.InfectionAlert.CDIFF] for k in sorted(monthly.keys())[-12:]]
+        cpes   = ['CPE'] + [monthly[k][models.InfectionAlert.CPE] for k in sorted(monthly.keys())[-12:]]
+        mrsas  = ['MRSA'] + [monthly[k][models.InfectionAlert.MRSA] for k in sorted(monthly.keys())[-12:]]
+        tbs    = ['TB'] + [monthly[k][models.InfectionAlert.TB] for k in sorted(monthly.keys())[-12:]]
+        vres   = ['VRE'] + [monthly[k][models.InfectionAlert.VRE] for k in sorted(monthly.keys())[-12:]]
+
+        context['overview_data'] = [ticks, cdiffs, cpes, mrsas, tbs, vres]
+
+        context['rfh_patients'] = UpstreamLocation.objects.filter(building='RFH').count()
+        context['rfh_siderooms'] = UpstreamLocation.objects.filter(building='RFH', room__startswith='SR').count()
+        context['weekly_alerts'] = models.InfectionAlert.objects.filter(
+            trigger_datetime__gte=today-datetime.timedelta(days=7)).count()
```

Comment:
I may be reading this wrong, do we not just care about RFH Infection alerts?

Reply:
> do we not just care about RFH Infection alerts?

Can you rephrase the question?
I don't know what you're asking :)